### PR TITLE
doc(setup-idea): consider the scenario of configuring authenticator in Java 11 for server

### PR DESCRIPTION
**Pull request by VGalaxies**
As title, inspired by @imbajin.

> EN doc will be synchronized later.

### CODE REVIEW

File: content/cn/docs/contribution-guidelines/hugegraph-server-idea-setup.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ weight: 4
 
 ### 背景
 
-在 [Quick Start](/docs/quickstart/hugegraph-server/) 部分已经介绍了使用**脚本**启停 HugeGraphServer 的流程。下面以 Linux 平台为例，介绍使用 **IntelliJ IDEA** 运行与调试 HugeGraph-Server 的流程。
+在 [Quick Start](/docs/quickstart/hugegraph-server/) 部分已经介绍了使用**脚本**启停 HugeGraphServer 的流程。下面以 Linux 平台为例，介绍使用 **IntelliJ IDEA** 运行与调试 HugeGraphServer 的流程。
```
Comment (imbajin) [inline content/cn/docs/contribution-guidelines/hugegraph-server-idea-setup.md:11]: e.g seems `HugeGraph-PD` is better than `HugeGraphPD` ?

File: content/cn/docs/contribution-guidelines/hugegraph-server-idea-setup.md [hunk 2]
```diff
@@ -116,7 +128,7 @@ curl "http://localhost:8080/graphs/hugegraph/graph/vertices" | gunzip
 
 #### 1. java: package sun.misc does not exist
 
-原因可能是在使用 Java 11 编译时触发了交叉编译，导致项目中使用的 `sun.misc.Unsafe` 找不到符号。有两种解决方案可供选择：
+原因可能是在使用 **Java 11** 编译时触发了交叉编译，导致项目中使用的 `sun.misc.Unsafe` 找不到符号。有两种解决方案可供选择：
 
 1. 在 IntelliJ IDEA 的 `Preferences/Settings` 中找到 `Java Compiler` 面板，然后关闭 `--release` 选项 (推荐)
 2. 或者将项目的 SDK 版本设置为 8
```
Comment (imbajin) [inline content/cn/docs/contribution-guidelines/hugegraph-server-idea-setup.md:134]:
```suggestion
2. 或者将项目的 SDK 版本设置为 8 (Deprecated soon)
```

File: content/cn/docs/contribution-guidelines/hugegraph-server-idea-setup.md [hunk 3]
```diff
@@ -74,6 +74,18 @@ rocksdb.wal_path=.
 - 将 `Main class` 设置为 `org.apache.hugegraph.dist.HugeGraphServer`
 - 设置运行参数为 `conf/gremlin-server.yaml conf/rest-server.properties`，同样地，这里的路径是相对于工作路径的，需要将工作路径设置为 `path-to-your-directory`
 
+> 若在 **Java 11** 环境下为 HugeGraphServer 配置了**用户认证** (authenticator)，则需要参考脚本启动时的[配置](https://github.com/apache/incubator-hugegraph/blob/master/hugegraph-server/hugegraph-dist/src/assembly/static/bin/hugegraph-server.sh#L124)，添加下述 **VM options**:
```
Comment (imbajin) [inline content/cn/docs/contribution-guidelines/hugegraph-server-idea-setup.md:77]:
```suggestion
> 若在 **Java 11** 环境下为 HugeGraphServer 配置了**用户认证** (authenticator)，在`InitStore` 和启动时应参考二进制包的脚本[配置](https://github.com/apache/incubator-hugegraph/blob/master/hugegraph-server/hugegraph-dist/src/assembly/static/bin/hugegraph-server.sh#L124)，添加下述 **VM options**:
```

**Review by imbajin — COMMENTED** [review]
THX

File: content/cn/docs/contribution-guidelines/hugegraph-server-idea-setup.md (hunk 3 above)
Comment (VGalaxies) [reply content/cn/docs/contribution-guidelines/hugegraph-server-idea-setup.md]: Due to the incomplete consistency of the two added VM options, separate writing for better clarity.

File: content/cn/docs/contribution-guidelines/hugegraph-server-idea-setup.md [hunk 4]
```diff
@@ -50,6 +50,18 @@ rocksdb.wal_path=.
 - 将 `Main class` 设置为 `org.apache.hugegraph.cmd.InitStore`
 - 设置运行参数为 `conf/rest-server.properties`，这里的路径是相对于工作路径的，需要将工作路径设置为 `path-to-your-directory`
 
+> 若在 **Java 11** 环境下为 HugeGraph-Server 配置了**用户认证** (authenticator)，需要参考二进制包的脚本[配置](https://github.com/apache/incubator-hugegraph/blob/master/hugegraph-server/hugegraph-dist/src/assembly/static/bin/init-store.sh#L52)，添加下述 **VM options**:
+>
+> ```text
```
Comment (imbajin) [inline content/cn/docs/contribution-guidelines/hugegraph-server-idea-setup.md:55]: maybe use bash or java keyword for it next time

**Review by imbajin — APPROVED** [review]

**Pull request merged**